services/istio/1.11.6/defaults/cm.yaml

```diff
@@ -24,3 +24,6 @@ data:
       serviceMonitor:
         labels:
           prometheus.kommander.d2iq.io/select: "true"
+    global:
+      image: "${kubetoolsImageRepository:=bitnami/kubectl}"
+      tag: "${kubetoolsImageTag:=1.23.5}"
```

Comment:
The default is 1.22.6: https://github.com/mesosphere/charts/blob/c7a2ddf604907d408f0fedcc41dd44af3693191d/staging/istio/values.yaml#L1-L3

Reply:
where is `kubetoolsImageRepository` set?

Reply:
I looked a little and couldn't find where we set it. But we already reference it a lot.

Reply:
I can't find it anywhere, I thought maybe it would be in k-cli in one of the substitution var files but I can't find it there either. maybe we're not actually setting it?? cc @mikolajb do you know 😅

Reply:
I can't find it anywhere, I thought maybe it would be in k-cli in one of the substitution var files but I can't find it there either. maybe we're not actually setting it?? cc @mikolajb do you know 😅

Reply:
Yeah, it looks like we don't set it 😮 
https://github.com/search?q=org%3Amesosphere+kubetoolsImageRepository&type=code

Reply:
_Lost in translation_ looks like - https://github.com/mesosphere/kommander/blob/v2.0.0/chart/kommander-bootstrap/templates/helmReleases/common/kommander-vars.yaml#L28-L29

Reply:
We should probably get that added back in :) https://github.com/mesosphere/kommander-cli/blob/ab8e8b2a3287265f8d05a3957dcd8950626e1d70/pkg/installer/flux_config.go#L193

Reply:
Created https://jira.d2iq.com/browse/D2IQ-87946 to track ^